File: src/1309-Decrypt-String-from-Alphabet-to-Integer-Mapping/1309.go
func freqAlphabets(s string) string {
    res := []byte{}
    for i := 0; i < len(s); i++ {
        if i + 2 < len(s) && s[i + 2] == '#' {
            res = append(res, 'j' + (s[i] - '1') * 10 + (s[i+1] - '0'))
            i += 2
        } else {
            res = append(res, 'a' + (s[i] - '1'))
        }
    }
    return string(res)
}

File: src/1309-Decrypt-String-from-Alphabet-to-Integer-Mapping/1309.java
class Solution {
    public String freqAlphabets(String s) {
        String res = "";
        for (int i = 0; i < s.length(); i++) {
            if (i + 2 < s.length() && s.charAt(i + 2) == '#') {
                res += (char)(Integer.parseInt(s.substring(i, i + 2)) + 96);
                i += 2;
            } else res += (char)(s.charAt(i) - '1' + 97);
        }
        return res;
    }
}